### PR TITLE
Added Hero Banner Section

### PR DESCRIPTION
**Pull request by Sukhendra523**
## Changes: 

- Added Body Wrapper
- Added Hero Banner Section
- Added HeroBgAnimation
- Added StarsCanvas to Hero BG

### CODE REVIEW

File: src/components/BodyWrapper/index.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const BodyWrapper = ({children}) => {
+  return (
+    <Body>
+        {children}
+    </Body>
+  )
+}
+
+export default BodyWrapper
```
Comment (Sukhendra-edcast) [inline src/components/BodyWrapper/index.jsx]: Add line break here

Comment (Sukhendra523) [reply src/components/BodyWrapper/index.jsx]: sure

**Review by Sukhendra-edcast — DISMISSED** [review]

File: src/components/HeroBgAnimation/index.js [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+        >
+          <stop stopColor="#13ADC7" />
+          <stop offset="1" stopColor="#13ADC7" stopOpacity="0" />
+        </linearGradient>
+      </defs>
+    </svg>
+  </Div>
+);
+
+export default HeroBgAnimation;
```
Comment (Sukhendra-edcast) [inline src/components/HeroBgAnimation/index.js]: Add line break here

File: src/components/Sections/BannerSection/RightContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            <Img src={HeroImg} alt="Rishav Chanda" />
+        </Tilt>
+        </motion.div>
+       
+        
+    </RightContainerDiv>
+  )
+}
+
+export default RightContainer
```
Comment (Sukhendra-edcast) [inline src/components/Sections/BannerSection/RightContainer.jsx]: Add Line Break Here

**Review by Sukhendra-edcast — DISMISSED** [review]

**Review by Sukhendra-edcast — APPROVED** [review]

**Pull request merged**